cdap-ui/app/features/adapters/templates/tabs/metrics.html

```diff
@@ -0,0 +1,34 @@
+<div ng-controller="AdaptersDetailMetricsController">
+  <div class="table-responsive">
```

Comment:
Here as well. When there are no metrics this tab should not be showing an empty table with just header.